cpp/src/parquet/statistics.cc

```diff
@@ -613,6 +615,8 @@ class TypedStatisticsImpl : public TypedStatistics<DType> {
     if (has_min_max) {
       PlainDecode(encoded_min, &min_);
       PlainDecode(encoded_max, &max_);
+      statistics_.is_min_value_exact = true;
```

Comment:
From the comment above (`Create stats from a thrift Statistics object`), shouldn't we expand the function parameters for these two fields?